btree/btree.c

```diff
@@ -1455,9 +1455,7 @@ static bool node_verify(const struct nd *node)
 
 #ifndef __KERNEL__
 /**
- * This function should get called in node_lock or tree_lock mode. As, it deals
- * with header and n_be_node_valid flag which will get updated in node_fini()
- * and node_free() respectively in tree_lock or node_lock mode only.
+ * This function should get called in node_lock mode only.
```

Comment:
Replace the comment with:  This function should be called after acquiring node lock.